lib/AST/Module.cpp

```diff
@@ -2020,6 +2020,10 @@ bool ModuleDecl::isStdlibModule() const {
   return !getParent() && getName() == getASTContext().StdlibModuleName;
 }
 
+bool ModuleDecl::isConcurrencyModule() const {
+  return !getParent() && getName() == getASTContext().getIdentifier("_Concurrency");
+}
```

Comment:
There's something like this already, albeit as a static in lib/AST/Module.cpp:

```
static bool isConcurrencyModule(DeclContext *dc) {
  auto *m = dc->getParentModule();
  return !m->getParent() && m->getName() == m->getASTContext().Id_Concurrency;
}
```

Reply:
Seems pretty recent and one use only, I'll converge on the new one since it's next to the isStdlib... one nicely, but take the Id_Concurrency :+1:

Thanks!